wpimath/src/main/native/cpp/apriltag/ApriltagFieldLayout.cpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#include "frc/apriltag/ApriltagFieldLayout.h"
+
+#include <wpi/json.h>
+#include "frc/geometry/Pose3d.h"
+#include "units/angle.h"
+#include "units/length.h"
+
+using namespace frc;
+
+frc::Pose3d ApriltagFieldLayout::GetTagPose(int id) const {
+    auto returnPose = Pose3d{};
+    for(Apriltag tag : m_apriltags) {
+        if(tag.id == id) {
+            returnPose = tag.pose;
+        }
+    }
+    if(m_mirror) {
+        returnPose = returnPose.RelativeTo(Pose3d{units::inch_t{54}, units::inch_t{27}, returnPose.Z(), Rotation3d{units::radian_t{0}, units::radian_t{0}, units::radian_t{180}}});
```

Comment:
```suggestion
        returnPose = returnPose.RelativeTo(Pose3d{54_ft, 27_ft, 0_ft, Rotation3d{0_deg, 0_deg, 180_deg}});
```
I'm pretty sure the Z component should be zero since the origins of both coordinate systems is on the ground.

Reply:
Did not realize how cool the units library was.  I'm pretty sure I caught this problem in Java and just forgot when I got over to C++.